### ISSUE TITLE
Ro-crates need a stanza pointing to the meta-data ro-crate when ready

### ISSUE THREAD

**Issue by cymon** [emo-bon/metagoflow-data-products-ro-crate#6]
Ro-crates need a stanza point to the meta-data ro-crate when ready

TODO

**Comment by kmexter** [comment]
Is this still a todo - not quite sure what you mean here? did you mean to the logsheets?

**Comment by cymon** [comment]
Yeah, I'm still a bit confused about this a well; even after the extensive going over [the ro-crate format had just recently](https://docs.google.com/document/d/1Uc2M_-tDBrQWJ5oO1FTAh4FEVc6jhh1NuS4PPzOBcOM/edit?tab=t.vxc7igwcuar8#heading=h.m4agfh9jik30) there still seems to be nowhere in the ro-crate that points to the sampling event metadata (the google "sampling" and "measured" logsheets data for the event). Or do people find it through the ENA Accession number, via the BioSamples Accession number only?

Or does it point to the sampling metadata ro-crate that Marc/Bram are working on?

**Comment by cymon** [comment]
Marc was talking about:

```
        "pcdm:hasMember":[
            {"@id": "{ena_accession_number_url}"},  
            {"@id": "{metagoflow_version}"}  
        ],
```
which we now have; so is that it? People have to through the ENA acc num to get to the sampling event data?

Note that Marc also wanted the actual sequence data files as part of the ./ hasPart (direct links to the raw data) - this is currently not implemented because I'm not sure it's even possible.

**Comment by kmexter** [comment]
well then, let's wait until @marc-portier comes back

**Comment by cymon** [comment]
Links to sample metadata are provided by hasPart in ./

            {"@id": "https://data.emobon.embrc.eu/sampling/{source_mat_id}"},
            {"@id": "https://data.embon.embrc.eu/sequence/{ref_code}"},

There are no descriptive stanza's, only the hasPart. Marc says they are not needed, but it might also not be clear what these links are without them - it trivial to add them, so should we for completeness? @marc-portier
